apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       jobKind: CronJob
     job:
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:prod-7cc7082-20230302152355 #{"$imagepolicy": "flux-system:fpl-cron-ccd-data-migration-tool"}
+      image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:pr-101-cc746c7-20230317112724 #{"$imagepolicy": "flux-system:fpl-cron-ccd-data-migration-tool"}
```

Comment:
a pr-image in production?

Reply:
CI check will fail with current request of` pr-*` image in production

Reply:
Updated to prod image - prod-7cc7082-20230302152355